acme/setup.py

```diff
@@ -75,6 +74,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=install_requires,
+    tests_require='mock<1.1.0' if sys.version_info < (2, 7) else 'mock',
```

Comment:
if we're really going with this approach (see https://github.com/letsencrypt/letsencrypt/pull/1665/files#r49645236), then please use a `tests_require` variable just as we do for all other package lists